config/locales/en.yml

```diff
@@ -226,8 +226,8 @@ en:
         one: Transparency data
         other: Transparency data
       world_location_news_article:
```

Comment:
We can remove this completely now can't we? And from all the translation files.
I believe the history partial was the only thing using it.